Упорядоченный список.py

```diff
@@ -0,0 +1,13 @@
+# Дан список целых чисел. Требуется переместить все ненулевые
+# элементы в левую часть списка, не меняя их порядок,
+# а все нули - в правую часть.
+user_input = input('Введите элементы списка: ').split()
+zero = 0
+lenn = len(user_input)
+for i in range(lenn):
+    if int(user_input[i]) == 0:
+        zero += 1
+for i in range(zero):

```

Comment:
you should use only one for loop according to task description

Reply:
rdy